OptimizelySDK/Optimizely.cs

```diff
@@ -51,17 +51,8 @@ public class Optimizely
 
         public static String SDK_VERSION {
             get {
-                // Example output: "1.2.1" .  Should be kept in synch with NuGet package version.
-#if NET35
-                Assembly assembly = Assembly.GetExecutingAssembly();
-#else
-                Assembly assembly = typeof(Optimizely).GetTypeInfo().Assembly;
-#endif
-                // Microsoft    Major.Minor.Build.Revision
-                // Semantic     Major.Minor.Patch
-                Version version = assembly.GetName().Version;
-                String answer = String.Format("{0}.{1}.{2}", version.Major, version.Minor, version.Build);
-                return answer;
+                // Should be kept in synch with NuGet package version.
+                return "2.0.0-beta1";
```

Comment:
What was the reason for changing this? Keeping this in sync with the NuGet package version manually could be error-prone since it relies on us actually remembering to do it

Reply:
We are keeping in synch with the NuGet version this release.  Unfortunately, the code which I wrote previously which I replaced here is getting the version number from MS AssemblyVersion which is (1) actually ignorant of the NuGet version in the *.nuspec and (2) doesn't support "-beta1" .  We'd like to do better, but what was there wouldn't get "-beta1" to OPTIMIZELY.COM .